sns_ik_examples/src/ik_tests.cpp

```diff
@@ -172,10 +172,19 @@ void test(ros::NodeHandle& nh, double num_samples_pos, double num_samples_vel,
   for (uint i=0; i < num_joint_pos; i++) {
     for (uint j=0; j<ll.data.size(); j++) {
       q(j)=fRand(ll(j), ul(j));
-      q_delta(j)=getDeltaWithLimits(q(j), position_seed_delta, ll(j), ul(j));
+      q_delta(j)=getDeltaWithLimits(q(j), delta_position_seed_value, ll(j), ul(j));
```

Comment:
This only needs to be calculated if use_delta_position_seed is true. This can be moved a few lines down within that conditional block.